src/shared/components/DescriptionField.tsx

```diff
@@ -15,7 +20,11 @@ export const DescriptionField = (props: FieldProps) => {
     return <Typography variant="subtitle2">{description}</Typography>;
 
   return (
-    <>
+    <Box
+      bgcolor="rgb(236, 243, 252)"
+      p={isSmallScreen ? 1 : 2}
+      m={isSmallScreen ? 1 : 2}
+    >
```

Comment:
```suggestion
      style={{borderRadius: 5}}
    >
```

what do you say to a bit of rounded corners? makes the box more consistent:

![Screen Shot 2021-04-10 at 9 28 16 AM](https://user-images.githubusercontent.com/9192912/114242557-31f79d00-99df-11eb-9fd5-c32b55832e6c.png)